### PR TITLE
Updating setup documents

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [x] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Edited for style and formatting. Updated setup docs to match docs published in https://github.com/PowerShell/PowerShell/tree/master/docs/installation. The goal is to get these kind of docs out the source code repo and into the docs repo.

Reason(s) for not updating all version of documents
--------------------------------------------------
- [x] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by sdwheeler** [comment]
@SteveL-MSFT @zjalexander - Can one of you review this change and merge? I am trying to get docs like this out of the source repo and into the docs repo.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fsetup%2FPowerShell-Core-on-ARM.md)
[reference/docs-conceptual/core-powershell/vscode/using-vscode.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/core-powershell/vscode/using-vscode.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fcore-powershell%2Fvscode%2Fusing-vscode.md)
[.openpublishing.redirection.json](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/PowerShell-Core-on-ARM.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/PowerShell-Core-on-ARM.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | |

<a name="reference%2Fdocs-conceptual%2Fsetup%2FPowerShell-Core-on-ARM.md"></a>
### [reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md)
 - **Line 8**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/installing-powershell-core-on-linux.md.md#raspbian).
 - **Line 8**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/installing-powershell-core-on-linux.md.md#raspbian).
 - **Line 8**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/installing-powershell-core-on-linux.md.md#raspbian).
 - **Line 8**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/installing-powershell-core-on-linux.md.md#raspbian).
 - **Line 8**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/installing-powershell-core-on-linux.md.md#raspbian).

<a name="reference%2Fdocs-conceptual%2Fcore-powershell%2Fvscode%2Fusing-vscode.md"></a>
### [reference/docs-conceptual/core-powershell/vscode/using-vscode.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/core-powershell/vscode/using-vscode.md)
 - **Line 13**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md).
 - **Line 13**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md).
 - **Line 13**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md).
 - **Line 13**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md).
 - **Line 13**: **[Warning]** Invalid file link:(~/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C24%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201804242033565704-2355%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FPcPUFcRkppRS%2FowqZ4lZB35xVhNhP2LBoprTV%2FbJYU%3D&st=2018-04-24T20%3A39%3A40Z&se=2018-05-25T20%3A44%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/PowerShell-Core-on-ARM.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/PowerShell-Core-on-ARM.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/vscode/using-vscode.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/core-powershell/vscode/using-vscode.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C24%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201804242115009298-2355%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=K3QIwEzfz3cxygMAxqqPCDlPHMB6NkqZAF2ahgAqQlQ%3D&st=2018-04-24T21%3A20%3A55Z&se=2018-05-25T21%3A25%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md [hunk 1]
```diff
@@ -46,20 +50,18 @@ sudo apt-get install -y powershell
 pwsh
 ```
 
-After registering the Microsoft repository once as superuser, from then on, you just need to use `sudo apt-get upgrade powershell` to update it.
+As superuser, Register the Microsoft repository. From then on, you just need to use `sudo apt-get upgrade powershell` to update the installation.
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: Seems odd for `Register` to be capitalized here.  Also, it seems you're cleaning up the markdown with semantic linefeeds (awesome!) but missed this one.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md [hunk 2]
```diff
@@ -652,9 +594,8 @@ sudo apt-get install libunwind8 libicu55
 wget http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
 sudo dpkg -i libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
 
-# Download & Install PowerShell
-wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0/powershell_6.0.0-1.ubuntu.16.04_amd64.deb
-sudo dpkg -i powershell_6.0.0-1.ubuntu.16.04_amd64.deb
+# Install PowerShell
+dpkg -i powershell_6.0.2-1.ubuntu.16.04_amd64.deb
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: sudo not required?

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md [hunk 3]
```diff
@@ -664,44 +605,61 @@ pwsh
...
 # Start PowerShell
-./powershell-6.0.0-x86_64.AppImage
+./powershell-6.0.2-x86_64.AppImage
 ```
 
 ### Uninstallation - Kali
 
 ```sh
-sudo dpkg -r powershell-6.0.0-x86_64.AppImage
+dpkg -r powershell_6.0.2-1.ubuntu.16.04_amd64.deb
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: sudo not required?

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
+# Installing PowerShell Core on macOS
+
+PowerShell Core supports macOS 10.12 and higher. All packages are available on our GitHub [releases][] page. Once the package is installed,
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md]: semantic linefeed

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: oops. I will fix that.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md (hunk 3 above)
Comment (sdwheeler) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: I had the same question but sudo was not in the example in https://github.com/PowerShell/PowerShell/blob/master/docs/installation/linux.md#kali

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md (hunk 4 above)
Comment (sdwheeler) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md]: Will fix

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: Not sure what happened here but I will fix.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md [hunk 5]
```diff
@@ -50,7 +50,8 @@ sudo apt-get install -y powershell
 pwsh
 ```
 
-As superuser, Register the Microsoft repository. From then on, you just need to use `sudo apt-get upgrade powershell` to update the installation.
+As superuser, register the Microsoft repository. From then on, you just need to use
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: The second sentence should be on a separate line

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md [hunk 6]
```diff
@@ -32,76 +32,122 @@ Be noted that when using the ZIP archive, you won't get the prerequisites check
...
+
+Windows IoT already comes with Windows PowerShell which we will use to deploy PowerShell Core 6.
+
+- Create `PSSession` to target device
+
+  ```powershell
+  $s = New-PSSession -ComputerName <deviceIp> -Credential Administrator
+  ```
+
+- Copy the zip package to the device
```
Comment (joeyaiello) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md]: Should use ZIP consistently unless used as part of a file version in a code block.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md [hunk 7]
```diff
@@ -32,76 +32,122 @@ Be noted that when using the ZIP archive, you won't get the prerequisites check
...
+
+  ```powershell
+  # change the destination to however you had partitioned it with sufficient
+  # space for the zip and the unzipped contents
+  # the path should be local to the device
+  Copy-Item .\PowerShell-6.0.2-win-arm32.zip -Destination u:\users\administrator\Downloads -ToSession $s
+  ```
+
+- Connect to the device and expand the archive
```
Comment (joeyaiello) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md]: Can we make this a numbered list? Feels weird as bullets. Even H3s make a little more sense to me...

Unless this is a pattern I'm not aware of, in which case, ignore me.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md [hunk 8]
```diff
@@ -32,76 +32,122 @@ Be noted that when using the ZIP archive, you won't get the prerequisites check
...
 
 ### Offline Deployment of PowerShell Core
 
 1. Use your favorite zip utility to unzip the package to a directory within the mounted Nano Server image.
-1. Unmount the image and boot it.
-1. Connect to the inbox instance of Windows PowerShell.
-1. Follow the instructions to create a remoting endpoint using the [another instance technique](#executed-by-another-instance-of-powershell-on-behalf-of-the-instance-that-it-will-register).
+2. Unmount the image and boot it.
+3. Connect to the inbox instance of Windows PowerShell.
+4. Follow the instructions to create a remoting endpoint using the [another instance technique](#executed-by-another-instance-of-powershell-on-behalf-of-the-instance-that-it-will-register).
```
Comment (joeyaiello) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md]: Use quotes for "another instance technique" like you have below?

File: reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+# PowerShell Core on ARM
```
Comment (joeyaiello) [inline reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md]: When this location change gets merged to live, https://aka.ms/pscore6-arm needs to be updated to point to the new location. I've given permission for writers to edit that link in my absence (@SteveL-MSFT and @SandeepSutari also have permission there if anything goes awry).

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md (hunk 5 above)
Comment (sdwheeler) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: fixed

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md (hunk 6 above)
Comment (sdwheeler) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md]: Fixed this everywhere we reference "ZIP package" or we are talking about the package. Left it lowercase in filenames and where "zip" and "unzip" refer to action/commands.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md (hunk 7 above)
Comment (sdwheeler) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md]: This is now a numbered list.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md (hunk 8 above)
Comment (sdwheeler) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md]: fixed

**Comment by sdwheeler** [comment]
@SteveL-MSFT I in corporated your and Joey's changes. If it looks good, please approve and merge.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/PowerShell-Core-on-ARM.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/PowerShell-Core-on-ARM.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/vscode/using-vscode.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/core-powershell/vscode/using-vscode.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-setup/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C26%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201804261743392222-2355%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=L6Z5SBJeLZDUF6qWkB3LGM7fa9u47LSRcByM2SlZ3Rc%3D&st=2018-04-26T17%3A49%3A18Z&se=2018-05-27T17%3A54%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md (hunk 9 above)
Comment (joeyaiello) [reply reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md]: This is the only one as far as I'm aware, but I'll double check with @SteveL-MSFT when I get back.

**Pull request merged**